fe/fe-core/src/test/java/com/starrocks/planner/OlapTableSinkTest.java

```diff
@@ -123,7 +124,7 @@ public void testSinglePartition() throws UserException {
             result = partition;
         }};
 
-        OlapTableSink sink = new OlapTableSink(dstTable, tuple, Lists.newArrayList(2L));
+        OlapTableSink sink = new OlapTableSink(dstTable, tuple, Lists.newArrayList(2L), TWriteQuorumType.MAJORITY);
```

Comment:
You can use pesudo cluster to add some test cases for quorum by simulating failed BE.